CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to casper-js-sdk.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 2.7.2
+
+### Fixed
+
+- Fix for bundling all of the existing types defined in the library.
+
```

Comment:
You might want to mention here that you've changed it to allow mixed case keys?

Reply:
this was done in `2.7.1` :)